### ISSUE TITLE
duti 1.5.4 fails to build on macOS 10.14 Mojave

### ISSUE THREAD

**Issue by fxcoudert** [moretension/duti#31]
```
checking build system type... i386-apple-darwin18.0.0
checking host system type... i386-apple-darwin18.0.0
checking target system type... i386-apple-darwin18.0.0
checking which SDK to use... configure: error: darwin18.0.0 is not a supported system
```

**Comment by aderion** [comment]
have same issue

**Comment by chdiza** [comment]
At first glance, this should be fixable in the same way that #20 was.

**Comment by chdiza** [comment]
I tried it; that same method does let `duti` build.

**Comment by moretension** [comment]
Merged PR #32
